### DISCUSSION TITLE
Sourcehut support

### DISCUSSION THREAD

**Discussion by bryanhonof** [magit/forge#370] [Ideas]
Hello!

I've recently found out about this project, and since I'm already using magit quite heavily, this just seemed to fit right in :).
Now, for github, and gitlab, this tool is already doing wonders. But I was wondering if support for [sourcehut](https://sourcehut.org/) is somewhere in the pipeline? And if not, I'd be willing to help out, but I'd need a bit of help getting started :).

**Comment by ktfleming** [comment]
I haven't used it myself but...isn't it already supported? See "Srht" under "2.3 Supported Semi-Forges" in [the manual](https://magit.vc/manual/forge.html#Supported-Semi_002dForges).

**Reply by bryanhonof** [reply]
Oh, I completely read over that, sorry!

**Comment by tarsius** [comment]
The plan is to support it better eventually but there is no timeline and it is unlikely I will work on it any time soon. I use sourcehut too now, so that might help motivate me to look into this a bit sooner.

**Reply by bryanhonof** [reply]
Is there perhaps a way I could help with that? My elisp experience is pretty limited, but I believe learning the hard way is a valid option :).

**Reply by tarsius** [reply]
It's quite a bit of work. Look at `forge-github.el`. You would basically have to do the same thing (minus notifications, probably).

**Comment by tarsius** [comment]
I still don't plan to add support for this, or another, forge any time soon. "Add support for more forges", is still a long time goal, but until I get there, there is no need to keep an issue/discussion per forge open.
